cmd/format.go

```diff
@@ -140,6 +141,18 @@ func formatStorageFlags() []cli.Flag {
 			Name:  "storage-class",
 			Usage: "the default storage class",
 		},
+		&cli.StringFlag{
+			Name:  "ca-certs",
```

Comment:
I think we could support these just like `insecure-skip-verify`, as a url query of bucket endpoint

Reply:
Well, I added them as flags to show them in the console in the same way most CLI out there do. But my mind is divided as they are not format-related params, they are transport-related

I checked the metadata store and i saw query params are there too, so if you prefer, I can parse them as query params, of course. If you confirm this, i will make proper changes, of course :)

Reply:
yes, we prefer query params

Reply:
What about current approach? :)